heudiconv/tests/conftest.py

```diff
@@ -0,0 +1,5 @@
+import os
+os.environ["GIT_AUTHOR_EMAIL"] = "maxm@example.com"
+os.environ["GIT_AUTHOR_NAME"] = "Max Mustermann"
+os.environ["GIT_COMMITTER_EMAIL"] = "maxm@example.com"
+os.environ["GIT_COMMITTER_NAME"] = "Max Mustermann"
```

Comment:
no need to define both AUTHOR and COMMITTER. Just define AUTHOR -- then 2nd one would default to it AFAIK if not set explicitly.

Reply:
nope, 2 tests still fail if I don't also set the committer, took me a while to figure that out as well...

https://ppb.chymera.eu/174ec2.log

Reply:
🤔 ok, let's see if fixture works

Reply:
it works :)